### PR TITLE
Fix continue watching carousel bug

### LINKED ISSUE #398: Continue watching carousel not updating with new videos
Continue watching carousel not updating with new videos as one watches more videos it doesn't add them to the Continue watching carousel.

Related to #282

### PR DESCRIPTION
**Pull request by drillprop**
Fix #398

### CODE REVIEW

**Review by eldiegod — APPROVED** [review]
Perfectly balanced, as all things should be

**Comment by drillprop** [comment]
I'll set it as a draft for now. Tried several approaches to fix  #393 also, but still did not manage to do this. Will look at it once more later with a fresh mind.

File: src/components/Sidenav/Sidenav.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ const Sidenav: React.FC<SidenavProps> = ({ items, isStudio }) => {
         <SidebarNavList>
           {items.map((item) => (
             <NavItem
-              key={item.name}
+              key={item.expandedName}
```
Comment (kdembler) [inline src/components/Sidenav/Sidenav.tsx]: Why is this change here?

File: src/api/client/cache.ts [hunk 2]
```diff
@@ -10,13 +10,14 @@ const getVideoKeyArgs = (args: Record<string, GetVideosQueryVariables['where']>
   const isPublic = args?.where?.isPublic_eq
   const channelIdIn = args?.where?.channelId_in ? JSON.stringify(args.where.channelId_in) : ''
   const createdAtGte = args?.where?.createdAt_gte ? JSON.stringify(args.where.createdAt_gte) : ''
+  const idIn = args?.where?.id_in ? JSON.stringify(args.where.id_in) : ''
```
Comment (kdembler) [inline src/api/client/cache.ts]: Why do we need that?

Comment (drillprop) [reply src/api/client/cache.ts]: We don't need that anymore, forgot to revert this

**Review by kdembler — APPROVED** [review]

**Pull request merged**